src/ViewHoldingsRecord.js

```diff
@@ -333,26 +333,14 @@ class ViewHoldingsRecord extends React.Component {
           <div data-test-holdings-view-page>
             <Pane
               defaultWidth={this.props.paneWidth}
+              appIcon={<AppIcon app="inventory" iconKey="holdings" />}
               paneTitle={
-                <div
-                  style={{ textAlign: 'center' }}
-                  data-test-header-title
-                >
-                  <AppIcon
-                    app="inventory"
-                    iconKey="holdings"
-                    size="small"
-                  />
-                  <strong>
-                    {holdingsRecord.permanentLocationId ? `${holdingsPermanentLocation.name} >` : null}
-                    {' '}
-                    {_.get(holdingsRecord, ['callNumber'], '')}
-                  </strong>
-                  &nbsp;
-                  <div>
-                    <FormattedMessage id="ui-inventory.holdings" />
-                  </div>
-                </div>
+                <span data-test-header-title>
+                  {holdingsRecord.permanentLocationId ? `${holdingsPermanentLocation.name} >` : null}
+                  {' '}
```

Comment:
_super_ nitpicky: should we move the space into the interpolated string on the line above? That would cause it to be omitted when there is no location, which seems like the correct behavior.

Reply:
I was thinking `... > ` (note the final trailing space). Not worth fighting about. I _hate_ ESLint's rules about whitespace, TBH, and feel they often do more harm then good.

Reply:
Ah.. 😅I missed that. Updating that now!